libHVCAENx527App/src/HVCAENx527.c

```diff
@@ -515,7 +515,7 @@ PDEBUG(3) printf( "DEBUG: number of parameters for each channel: %d\n", npar);
 					if( hvch->pplist == NULL)
 					{
 						printf( "InitCrate: Failed to calloc parameter property list.\n");
-						return;
+						continue;
 					}
```

Comment:
The error here is we have failed to allocate memory and hence initialisation has failed, it might be return is more appropriate?